### ISSUE TITLE
[Accessibility] SVGs missing tags to correctly be identified by screen readers

### ISSUE THREAD

**Issue by rpkelly** [bottlerocket-os/bottlerocket-project-website#116]
**Path to the page with the problem (_do not_ include domain name):** Entire site

**Describe the problem:** SVGs used throughout are missing tags to be correctly identified by common screen readers, and will either not be identified at all, or will simply be identified as "image".

**Screen Shot, if possible**

**Describe how the site should look?** See https://a11y-101.com/development/svg for how to handle SVGs with screen readers.
